clang/lib/CAS/IncludeTree.cpp

```diff
@@ -686,6 +694,12 @@ llvm::Error IncludeTree::Node::print(llvm::raw_ostream &OS, unsigned Indent) {
     return getIncludeTree().print(OS, Indent);
   case NodeKind::ModuleImport:
     return getModuleImport().print(OS, Indent);
+  case NodeKind::SpuriousImport: {
+    OS << "SpuriousImport\n";
+    if (auto Err = llvm::cantFail(getSpuriousImport().getModuleImport()).print(OS, Indent))
+      return Err;
+    return llvm::cantFail(getSpuriousImport().getIncludeTree()).print(OS, Indent);
```

Comment:
Why are these `cantFail`? Typically we would just return the Error if there is one.